xml/art_solution_automation.xml

```diff
@@ -166,6 +166,175 @@ https://confluence.suse.com/x/8IEnGw
+            </para>
+            <screen>&prompt.root;<command>mkdir</command> --parents /srv/pillar/{hana,netweaver,drbd}</screen>
+            <para>
+              The directory names match with the names in the <filename>top.sls</filename>
+              file (see next step).
+            </para>
+          </step>
+          <step>
+            <para>
+              Create the file <filename>/srv/pillar/top.sls</filename> with the following content:
+            </para>
+            <screen language="sls">base:
+    'hana01,hana02':
+        - hana.hana
+        - hana.cluster
+
+    'netweaver01,netweaver02':
+        - netweaver.netweaver
+        - netweaver.cluster
+
+    'drbd01,drbd02':
+        - drbd.drbd
+        - drbd.cluster</screen>
+          </step>
+        </substeps>
+      </step>
+      <step>
+        <para>
+          Download the pillar files and store them in the correct directory under <filename
+            class="directory">/srv/pillar</filename>:
+        </para>
+        <substeps>
+          <step>
+            <remark>toms 2020-09-23: You can download the files and store them
+            to the directory in one go. However, I thought due to security reasons,
+            it would be better to split this into two tasks. Is that okay?
+            Or is there an easier way to do it?
+            </remark>
+            <para>
+              Download the the examples from the upstream GitHub repository:
+            </para>
+            <screen>&prompt.user;<command>wget</command> -O hana.sls \
+  https://github.com/SUSE/saphanabootstrap-formula/raw/master/pillar.example
+&prompt.user;<command>wget</command> -O netweaver.sls \
+  https://github.com/SUSE/sapnwbootstrap-formula/raw/master/pillar.example
+&prompt.user;<command>wget</command> -O drbd.sls \
+  https://github.com/SUSE/drbd-formula/raw/master/pillar.example
+&prompt.user;<command>wget</command> -O cluster.sls \
+  https://github.com/SUSE/habootstrap-formula/raw/master/pillar.example</screen>
+          </step>
+          <step>
+            <para>
+              Copy all pillar files to the right directory:
+            </para>
+            <screen>&prompt.sudo;<command>cp</command> -vi hana.sls cluster.sls /srv/pillar/hana
+&prompt.sudo;<command>cp</command> -vi netweaver.sls cluster.sls /srv/pillar/netweaver
+&prompt.sudo;<command>cp</command> -vi drbd.sls cluster.sls /srv/pillar/drbd</screen>
+          </step>
+        </substeps>
+      </step>
+      <step>
+        <para>
+          Modify the content of the pillar files depending on your needed configuration
+          of each &salt; formula.
+        </para>
+        <para>
+          Notice, in this example, we have three different
+          <filename>cluster.sls</filename> files. Each might have different HA cluster
+          requirements.
+        </para>
+        <remark>toms 2020-09-23: Maybe we should link to the upstream Salt documentation here?
```

Comment:
I don't think referencing any salt documentation will help. Here the thing is that the users must know which are the things they need to configure (and that's the difficult part).
As we commented, we cannot give already done pillar files in the documentation. Maybe we could find other source, but in any case we will always have corner case.

Reply:
Right. It was more of an idea. I think, I can solve that by adding an additional paragraph *before* the procedure:

```
The following procedure is written in a general way to make it fit for all
use cases. You need some knowledge about &salt; and how to
customize it to your needs.
```